common/app/views/fragments/footer.scala.html

```diff
@@ -44,17 +44,29 @@
                 <li class="colophon__item"><a data-link-name="int : footer : email sign-up" href="@LinkTo{/world/2013/oct/04/1}">
                     daily email sign up</a></li>
                 <li class="colophon__item"><a data-link-name="int : footer : facebook" href="https://www.facebook.com/theguardian">
-                    Facebook</a></li>
```

Comment:
Minor comment but I would have opted for `text-transform: lowercase`, as this is purely style based. But this is fine :smile: :+1: